db/chores/migrate_messages_to_scheduled_messages.rb

```diff
@@ -0,0 +1,91 @@
+class MigrateMessagesToScheduledMessages
+  def perform
+    execute("SELECT * FROM onboarding_messages").each do |row|
+      migrate_onboarding_message_to_scheduled_message(row)
+    end
+    execute("SELECT * FROM quarterly_messages").each do |row|
+      migrate_quarterly_message_to_scheduled_message(row)
+    end
+  end
+
+  private
+
+  def migrate_onboarding_message_to_scheduled_message(row)
+    # Migrate to new table
+    row["type"] = 0
+    values = ["created_at","updated_at","title","body","days_after_start","time_of_day","type","end_date","deleted_at"].map do |column|
```

Comment:
small style thing: would it be possible to put each array val on its own line? I think that would be easier to read